src/verifyEmail.js

```diff
@@ -0,0 +1,27 @@
+function verifyEmail(appId, serverURL) {
```

Comment:
The email related routes are really grouped together in any logical way, so I'm not sure that makes sense.